src/gg.regression.unity.bots/Editor/Scripts/BotManagement/IRGBotFinder.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEngine;
+using System.Linq;
+using System.Collections.Generic;
+
+public class IRGBotFinder : Editor
+{
+    [DidReloadScripts]
+    private static void OnScriptsReloaded()
+    {
+        // Only run this method in the Unity editor, not at runtime
+        if (!EditorApplication.isPlayingOrWillChangePlaymode || EditorApplication.isPlaying)
+        {
+            return;
+        }
```

Comment:
Is there actually a downside of allowing this at play time? Wouldn't this have the benefit of letting us see new bots on the fly (assuming your game even does well with script reloading in the first place)

Reply:
@rcornwall-reg note that in our PR process, never resolve comments yourself - the reviewers will mark the comments as resolved. Instead, just put a comment saying that you fixed it.